opentelemetry-sdk/src/export/logs/mod.rs

```diff
@@ -91,6 +91,25 @@ pub trait LogExporter: Send + Sync + Debug {
         // By default, all logs are enabled
         true
     }
+
+    /// This is a hint to ensure that the export of any Logs the exporter
+    /// has received prior to the call to this function SHOULD be completed
+    /// as soon as possible, preferably before returning from this method.
+    ///
+    /// This function SHOULD provide a way to let the caller know
+    /// whether it succeeded, failed or timed out.
+    ///
+    /// This function SHOULD only be called in cases where it is absolutely necessary,
+    /// such as when using some FaaS providers that may suspend the process after
+    /// an invocation, but before the exporter exports the completed logs.
+    ///
+    /// This function SHOULD complete or abort within some timeout. This function can be
```

Comment:
Let's update the comments here to reflect the actual implementation instead of putting the spec guidelines for this method.

We could put something like:
```rust
/// This method will block the current thread until all pending log records are exported. If the export was not
/// successful, an error is returned.
```